src/python/foglamp/admin_api/README.rst

```diff
@@ -1,39 +1,86 @@
 Introduction
 ------------
```

Comment:
change `------------` to `^^^^^^^^^^^^^`  for all headings; in backend.rst we shall need to make file look like:

```
Backend
--------

Requirements:
^^^^^^^^^^^^^


* Python 3.5.2
* Ubuntu 16.0.4 or up
* Postgres 9.6.3


Installation:
^^^^^^^^^^^^^
....
```